.github/workflows/ci_windows.yml

```diff
@@ -0,0 +1,52 @@
+name: Run Windows CI
```

Comment:
Could this be merged into `.github/workflows/ci.yml` as a separate job?  Something like

```yaml
jobs:
  unixish:
    ... existing Linux/macOS setup ...
  windows:
    ... Windows CI setup ...
```

Reply:
I tried it. For some reason, msgpack is no longer found in the MINGW_32 build. Hmm, that's about it for today...